src/SDKs/DataLake.Store/Management.DataLake.Store/generate.cmd

```diff
@@ -6,8 +6,8 @@
 @echo off
 setlocal
 
-set accountSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0ade6547eff89bab427eb400ebc5da870f2c5bd0/arm-datalake-store/account/2016-11-01/swagger/account.json"
-set filesystemSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0ade6547eff89bab427eb400ebc5da870f2c5bd0/arm-datalake-store/filesystem/2016-11-01/swagger/filesystem.json"
+set accountSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/12a6c2299296b6877fa1e5a03ed42752445877dc/arm-datalake-store/account/2016-11-01/swagger/account.json"
```

Comment:
@shahabhijeet this has been fixed and I removed all other entries. I am also, in the next iteration, moving the package ID to the top for easier identification.